nominees/cloud-carbon-footprint.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "Cloud Carbon Footprint",
+  "aliases": [
+    "CCF"
+  ],
+  "description": "Cloud Carbon Footprint is an open source tool and methodology that provides visibility and tooling to measure, monitor and reduce organizations' infrastructure carbon emissions.",
+  "website": "https://www.cloudcarbonfootprint.org/",
+  "license": [
+    {
+      "spdx": "Apache-2.0",
+      "licenseURL": "https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/blob/trunk/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 7,
+      "evidenceText": "CCF provides estimates for both energy and carbon emissions for all types of cloud usage including embodied emissions from manufacturing, with the ability to drill down into emissions by cloud provider, account, service and time period.",
+      "evidenceURL": "https://www.cloudcarbonfootprint.org/"
+    },
+    {
+      "SDGNumber": 11,
+      "evidenceText": "Understanding your cloud carbon footprint baseline helps you make specific, targeted reductions to reduce your emissions, such as rightsizing, deleting idle instances, and more.",
```

Comment:
I am thinking [SDG 11](https://sdgs.un.org/goals/goal11) is a stretch. Doesn't have a clear alignment with the indicators for the goal, unless there is additional evidence withheld from this submission.

![Target 11.6 of SDG 11: "By 2030, reduce the adverse per capita environmental impact of cities, including by paying special attention to air quality and municipal and other waste management."](https://user-images.githubusercontent.com/4721034/154099950-52116bba-bd50-484f-a843-92d4f6071d1c.png "Target 11.6 of SDG 11: \"By 2030, reduce the adverse per capita environmental impact of cities, including by paying special attention to air quality and municipal and other waste management.\"")

![Target 11.a of SDG 11: "Support positive economic, social and environmental links between urban, per-urban and rural areas by strengthening national and regional development planning."](https://user-images.githubusercontent.com/4721034/154099998-386f4389-6e0b-40e7-92b6-849a64ee4134.png "Target 11.a of SDG 11: \"Support positive economic, social and environmental links between urban, per-urban and rural areas by strengthening national and regional development planning.\"")

Reply:
Resolved in commit ca0b894